File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/LabTestsLabTestBatcheLabTestTypesItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class LabTestsLabTestBatcheLabTestTypesItem {
    @JsonProperty("AlwaysPasses")
    public Boolean alwaysPasses;
    @JsonProperty("DependencyMode")
    public Integer dependencyMode;
    @JsonProperty("Id")
    public Integer id;
    @JsonProperty("InformationalOnly")
    public Boolean informationalOnly;
    @JsonProperty("LabTestResultExpirationDays")
    public Integer labTestResultExpirationDays;
    @JsonProperty("LabTestResultMaximum")
    public String labTestResultMaximum;
    @JsonProperty("LabTestResultMinimum")
    public String labTestResultMinimum;
    @JsonProperty("LabTestResultMode")
    public Integer labTestResultMode;
    @JsonProperty("MaxAllowedFailureCount")
    public Integer maxAllowedFailureCount;
    @JsonProperty("Name")
    public String name;
    @JsonProperty("RequiresTestResult")
    public Boolean requiresTestResult;
    @JsonProperty("ResearchAndDevelopment")
    public Boolean researchAndDevelopment;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/LabTestsLabTestBatcheLabTestTypesItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class LabTestsLabTestBatcheLabTestTypesItem(
    @JsonProperty("AlwaysPasses")
    val alwaysPasses: Boolean? = null,
    @JsonProperty("DependencyMode")
    val dependencyMode: Int? = null,
    @JsonProperty("Id")
    val id: Int? = null,
    @JsonProperty("InformationalOnly")
    val informationalOnly: Boolean? = null,
    @JsonProperty("LabTestResultExpirationDays")
    val labTestResultExpirationDays: Int? = null,
    @JsonProperty("LabTestResultMaximum")
    val labTestResultMaximum: String? = null,
    @JsonProperty("LabTestResultMinimum")
    val labTestResultMinimum: String? = null,
    @JsonProperty("LabTestResultMode")
    val labTestResultMode: Int? = null,
    @JsonProperty("MaxAllowedFailureCount")
    val maxAllowedFailureCount: Int? = null,
    @JsonProperty("Name")
    val name: String? = null,
    @JsonProperty("RequiresTestResult")
    val requiresTestResult: Boolean? = null,
    @JsonProperty("ResearchAndDevelopment")
    val researchAndDevelopment: Boolean? = null
)
